lib/tasks/import/log.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+desc 'Import daily log from Access CSV at data/daily_log.csv'
+namespace :import do
+  task log: :environment do
+    CSV.foreach('data/daily_log.csv', encoding: 'ISO-8859-1', headers: true).with_index 2 do |row, line|
+      date = Date.parse row.fetch('Date:')
+      user_name = row.fetch('Dispatcher:')
+      if user_name.is_a? String
+        last_name, first_name = user_name.split(', ')
+        user = User.find_by name: [first_name, last_name].join(' ')
```

Comment:
We can't import prior dispatchers - they're removed from the table (rather than set as inactive).

Your diagnosis is correct — these imports will result in a good deal of invalid data. Up to us to determine how to either correct the data or handle its invalidness.